akka-actor-typed/src/main/scala/akka/actor/typed/internal/PropsImpl.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed.internal
+
+import akka.actor.typed.DispatcherSelector
+import akka.actor.typed.Props
+import akka.annotation.InternalApi
+
+/**
+ * INTERNAL API
+ */
+@InternalApi private[akka] object PropsImpl {
+
+  /**
+   * The empty configuration node, used as a terminator for the internally linked
+   * list of each Props.
+   */
+  case object EmptyProps extends Props {
+    override def next = throw new NoSuchElementException("EmptyProps has no next")
+    override def withNext(next: Props): Props = next
+  }
+
+  final case class DispatcherDefault(next: Props) extends DispatcherSelector {
+    override def withNext(next: Props): Props = copy(next = next)
+  }
+  object DispatcherDefault {
+    // this is hidden in order to avoid match on this object
+    private val empty = DispatcherDefault(EmptyProps)
+
+    /**
+     * Retrieve an instance for this configuration node with empty `next` reference.
+     */
+    def apply(): DispatcherDefault = empty
```

Comment:
Is there still a reason to keep the `apply` now that the entire thing is internal?

Reply:
ok, removed in fed7099